getting-started.md

```diff
@@ -0,0 +1,71 @@
+
+### What I do
+
+Please provide a brief summary of what you do and who you do it for. Openness in our community is helpful in developing a strong network. Knowing where someone works or their job role increases their credibility within the community. You do not _need_ to provide your employer's name, but it does help increase credibility.
+
+Please see the below examples:
+
+* Junior Developer at Northern Trail Outfitters
+* Technical Architect at Salesforce
+* Business Analyst at System Integrator X
+
+### Profile photo
+
+Please upload a unique Profile photo or avatar as they are sometimes called. This will allow members of the community to easily recognize your posts and give you a unique persona within the community. If you don't provide one, your default Profile photo will be a Slack-generated image. However, there are only so many variations of this image and it becomes very monotonous to see the same patterns all over the place.
+
+Example of the Slack default Profile photo:
+
+![Sample Slack Profile Photo](assets/default-slack-avatar.png)
+
+## Dos and Don'ts
+
+### Do ask questions
+
+There is a [`#newbies`](https://sfcc-unofficial.slack.com/messages/CDG4R5CT1) channel that is a good place for asking questions that don't appear to be relevant for any other channel. Questions that aren't related to newcomers can be asked in [`#general`](https://sfcc-unofficial.slack.com/messages/CAT794PC3). _See also:_ [Do Respect Channel Topics](#do-respect-channel-topics).
+
+### Do respect others
+
+As a community based around a professional topic and setting, respect is a critical value for the success of the community overall. Remember the _Golden Rule_, "Treat others as you would wish to be treated."
+
+### Do respect channel topics
+
+Many channels exist within the community. Some have specific topics such as `#ocapi`, `#einstein`, and `#sfcc-certification`. Others, such as `#general` and `#random` don't have a specific topic. Please try to locate the right place to post your message.
+
+### Do create public channels
+
+If a channel that you feel should exist doesn't exist, go ahead and create it. Keep in mind that there may not be a significant number of community members that may be interested in that topic. "If you build it, they will come" doesn't always apply here. So think about the amount of people who would be interested in that topic before creating a channel dedicated to it.
+
+### Do exercise caution when creating private channels
+
+Private channels can only be converted to a public channel by an administrator. So try to avoid creating them unless you really need to have a small private group. One such legitimate use case is `admin-chat` where the administrators discuss issues relating to the management of the Slack community.
```

Comment:
Actually from what I know once a channel is private it can never be made public again (due to message encryption). At least that was the case when I wanted to make a private channel public on our internal Slack some time ago.

Reply:
You are correct. At some point in the past you could convert them... it seems now that is no longer possible.